libcodechecker/session_manager.py

```diff
@@ -130,26 +130,26 @@ def check_file_owner_rw(file_to_check):
     return True
 
 
-def load_session_cfg(session_cfg_file):
+def load_server_cfg(server_cfg_file):
     """
     Tries to load the session config file which should be a
     valid json file, if loading fails returns an empty dict.
     """
 
     scfg_dict = {}
     try:
-        with open(session_cfg_file, 'r') as scfg:
+        with open(server_cfg_file, 'r') as scfg:
```

Comment:
We are having more and more JSON files to store configurations. I wonder if we do have a utility that loads such file and handle the errors somewhere or in case we don't it might be worth to have one and reduce code duplication.

Reply:
This isn't "more and more" file, the file just got renamed and added some extra configuration variables. My staled patch, #1172, contains a utility function that allows you to load a JSON file and get a default dict if the loading failed.